File: hw/palomo/u3/hw20FarmAnimalSystem/FarmAnimalSystem/src/main/java/ec/espe/edu/farmanimalsystem/model/Cut.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.espe.edu.farmanimalsystem.model;

/**
 *
 * @author Cristian
 */
public class Cut {
    private int id;
    private String description;
    private String Procedure;
    private float weight;

    public Cut(int id, String description, String Procedure, float weight) {
        this.id = id;
        this.description = description;
        this.Procedure = Procedure;
        this.weight = weight;
    }
    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }

    /**
     * @return the Procedure
     */
    public String getProcedure() {
        return Procedure;
    }

    /**
     * @param Procedure the Procedure to set
     */
    public void setProcedure(String Procedure) {
        this.Procedure = Procedure;
    }

    /**
     * @return the weight
     */
    public float getWeight() {
        return weight;
    }

    /**
     * @param weight the weight to set
     */
    public void setWeight(float weight) {
        this.weight = weight;
    }

    @Override
    public String toString() {
        return "Cut{" + "id=" + id + ", description=" + description + ", Procedure=" + Procedure + ", weight=" + weight + '}';
    }    
}

File: hw/tipantiza/u3/hw21FarmSystemPOO/FarmSystem_py/model/cut.py
class Cut:
    def __init__(self, id: int, description: str, procedure: str, weight: float):
        self.id = id
        self.description = description
        self.procedure = procedure
        self.weight = weight
    
    def __str__(self):
        return f"Cut(id={self.id}, description={self.description}, procedure={self.procedure}, weight={self.weight})"
    
    def get_id(self) -> int:
        return self.id
    
    def set_id(self, id: int):
        self.id = id
    
    def get_description(self) -> str:
        return self.description
    
    def set_description(self, description: str):
        self.description = description
    
    def get_procedure(self) -> str:
        return self.procedure
    
    def set_procedure(self, procedure: str):
        self.procedure = procedure
    
    def get_weight(self) -> float:
        return self.weight
    
    def set_weight(self, weight: float):
        self.weight = weight